packages/core/core-flows/src/order/workflows/order-edit/request-order-edit.ts

```diff
@@ -153,12 +152,6 @@ export const requestOrderEditRequestWorkflow = createWorkflow(
     const updateOrderChangesData = getOrderChangesData({ input, orderChange })
     updateOrderChangesStep(updateOrderChangesData)
 
-    createOrUpdateOrderPaymentCollectionWorkflow.runAsStep({
```

Comment:
wait, when does this happen now ? :D

Reply:
We run this when OE request is confirmed because, now, this workflow might include payment cancellation and we don't want to do that if OE ends up being declined or canceled.